tests/test_fixtures/rule_based_profiler/example_notebooks/MultiBatchExample_SqlExample.ipynb

```diff
@@ -101,9 +93,7 @@
    "id": "df19a29c",
    "metadata": {},
    "source": [
-    "In our example, we add a `SimpleSqlalchemyDatasource` named `taxi_multi_batch_sql_datasource` with 2 `tables`, namely `yellow_tripdata_sample_2020_01`, `yellow_tripdata_sample_2020_02`. The configuration for `yellow_tripdata_sample_2020_02` is mostly used for our `Checkpoint` at the end, so the following doc will focus more on `yellow_tripdata_sample_2020_01`.\n",
-    "\n",
-    "The configuration for `yellow_tripdata_sample_2020_01` also contains 2 `partitioners` which correspond to names of `ConfiguredAssetSqlDataConnectors`. \n",
+    "In our example, we add a `SimpleSqlalchemyDatasource` named `taxi_multi_batch_sql_datasource` with 1 table.The configuration for `yellow_tripdata_sample_2020` also contains 2 `partitioners` which correspond to names of `ConfiguredAssetSqlDataConnectors`. \n",
```

Comment:
Missing a space between the 1st and 2nd sentence.

Reply:
Is `SimpleSqlalchemyDatasource` preferred? I thought we were trying to get away from promoting it.

Reply:
Thank you for the question @NathanFarmer currently we haven't made a decision between certifying the `Datasource` vs using `SimpleSqlDatasource`. In order to use splitters properly using the current state of the code I believe we have to use `SimpleSqlDatasource`

Reply:
🙇🏻